lib/request.js

```diff
@@ -14,31 +16,34 @@ var Request = function(hostname, headers, logLevel) {
   }
 
   this.hostname = hostname;
-  this.headers = headers
+  this.headers = headers;
   this.logger = logger;
   logger.level = logLevel || 0;
+  this.failOnLimitReached = typeof failOnLimitReached === 'undefined' ?
```

Comment:
If `false` is the fallback when the variable is undefined, I think `!!failOnLimitReached` would work here too?